src/Core/Install.php

```diff
@@ -0,0 +1,340 @@
+<?php
+/**
+ * @file src/Core/Install.php
+ */
+namespace Friendica\Core;
+
+use Friendica\BaseObject;
+use Friendica\App;
+use Friendica\Database\DBStructure;
+use Friendica\Object\Image;
+use Friendica\Util\Network;
+
+use Exception;
+use DOMDocument;
+
+/**
+ * @brief Contains the class with install relevant stuff *
+ */
+class Install extends BaseObject
+{
+	public static function setInstallMode()
+	{
+		self::getApp()->mode = App::MODE_INSTALL;
+	}
+
+	public static function check($phpath = 'php')

```

Comment:
Please add A DocBlock comment for this function, including a precise description of the returned data structure.